utils/renderTemplate.ts

```diff
@@ -41,9 +41,13 @@ function renderTemplate(src, dest) {
     return
   }
 
-  if (filename.startsWith('_')) {
-    // rename `_file` to `.file`
-    dest = path.resolve(path.dirname(dest), filename.replace(/^_/, '.'))
+  if (filename === '.gitignore') {
```

Comment:
Please retain the renaming logic.
Because `.gitignore` won't be packed by npm, we have to use `_gitignore` in templates.